controllers/infra/common/common.go

```diff
@@ -3,28 +3,28 @@ package common
 import "time"
 
 const (
-	InfraInstancesLabel  = "infra.sealos.io/instances/label"
-	InfraInstancesUUID   = "infra.sealos.io/infra/uuid"
-	InfraInstancesIndex  = "infra.sealos.io/instances/index"
-	InfraVolumesLabel    = "infra.sealos.io/volumes/label"
-	IPTypePublic         = "public"
-	IPTypePrivate        = "private"
-	DataVolumeLabel      = "data"
-	RootVolumeLabel      = "root"
-	TryTimes             = 8
-	TrySleepTime         = time.Second
-	TRUELable            = "true"
-	SealosInfraFinalizer = "infra.sealos.io/finalizers"
-	InfraSecretPrefix    = "infra-secret"
-	InfraVolumeIndex     = "infra.sealos.io/volumes/index"
-	VolumeInfraID        = "infra.sealos.io/volumes/infraID"
-	InstanceState        = "instance-state-name"
-	DefaultRegion        = "cn-north-1b"
-	KeyPairUser          = "sealos.io/aws/keypair/user"
-	KeyPairGeneral       = "sealos.io/aws/keypair/general"
-	KeyPairRoot          = "sealos.io/aws/keypair/root"
-	ArchAmd64            = "amd64"
-	ArchArm64            = "arm64"
+	InfraInstancesLabel   = "infra.sealos.io/instances/label"
+	InfraInstancesUUID    = "infra.sealos.io/infra/uuid"
+	InfraInstancesIndex   = "infra.sealos.io/instances/index"
+	InfraVolumesLabel     = "infra.sealos.io/volumes/label"
+	IPTypePublic          = "public"
+	IPTypePrivate         = "private"
+	DataVolumeLabel       = "data"
+	RootVolumeLabel       = "root"
+	TryTimes              = 8
+	TrySleepTime          = time.Second
+	TRUELable             = "true"
+	SealosInfraFinalizer  = "infra.sealos.io/finalizers"
+	InfraSecretPrefix     = "infra-secret"
+	InfraVolumeIndex      = "infra.sealos.io/volumes/index"
+	VolumeInfraID         = "infra.sealos.io/volumes/infraID"
+	InstanceState         = "instance-state-name"
+	DefaultRegion         = "cn-north-1b"
+	KeyPairUser           = "sealos.io/keypair/user"
+	ArchAmd64             = "amd64"
+	ArchArm64             = "arm64"
+	InstanceStatusRunning = "running"
+	AliyunKeyPairPrefix   = "infra"
```

Comment:
KeyPairePrefix is "infra"?

Reply:
> KeyPairePrefix is "infra"?

because keypair name which begin with number 0-9 is invalid in Aliyun platfrom, so it needs a prefix begin with english character, whatever is okay but for simple i choose "infra"